package.json

```diff
@@ -45,15 +45,15 @@
     "bindings": "^1.5.0",
     "eventemitter2": "^5.0.1",
     "nan": "^2.15.0",
-    "prebuild-install": "^6.1.4"
+    "prebuild-install": "^7.0.1"
   },
   "devDependencies": {
     "chai": "^4.1.2",
     "chalk": "^2.4.1",
     "eslint": "^4.19.1",
     "jasmine": "^3.1.0",
-    "node-abi": "^3.1.0",
-    "node-gyp": "^8.2.0",
-    "prebuild": "^11.0.0"
+    "node-abi": "^3.8.0",
+    "node-gyp": "^8.4.0",
+    "prebuild": "^11.0.3"
```

Comment:
When we try to `Prebuild binaries for all ABIs`, the following error occurs:

https://github.com/MadLittleMods/node-usb-detection/runs/5400425223?check_suite_focus=true
```
/tmp/prebuild/node/17.0.0/include/node/v8-persistent-handle.h:10:10: fatal error: v8-weak-callback-info.h: No such file or directory
```

This is tracked on the `prebuild` side in https://github.com/prebuild/prebuild/issues/284 and was fixed in Node.js v17.0.1, see https://github.com/nodejs/node/pull/40526

Reply:
We can follow https://github.com/serialport/node-serialport/pull/2356 and drop v17 support for now ⏩

Reply:
Followed-up in https://github.com/MadLittleMods/node-usb-detection/pull/165#discussion_r823362667

Side-effect of not many prebuilds because only building for 3 versions instead of all of the in between versions 😢

Reply:
We might be able to try the alternative workaround in https://github.com/prebuild/prebuild/issues/284#issuecomment-1085490595 to get all of the versions supported 😮